newIDE/app/src/ObjectsList/index.js

```diff
@@ -1480,6 +1497,12 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
             ),
           },
           { type: 'separator' },
+          {
+            label: i18n._(t`Swap asset`),
```

Comment:
I wonder if this should be:
```suggestion
            label: i18n._(t`Swap assets`),
```

To avoid creating the confusion between "asset" and "object". In GDevelop, an "asset" from the asset store is typically an object... but generally speaking assets are the images, resources, inside an object.